Sources/HaCWebsiteLib/ViewModels/EventStream.swift

```diff
@@ -0,0 +1,22 @@
+import HaCTML
+
+// swiftlint:disable line_length
+
+struct EventStream {
```

Comment:
This is more accurately named "LandingUpdateFeed". If we'd like to make this general-purpose for anywhere we use an Update Feed, we'll want to strip the word `Landing` from the CSS classes below and in the stylesheets